### DISCUSSION TITLE
The unit for snowLevel

### DISCUSSION THREAD

**Discussion by kelortondo** [weather-gov/api#206] [Q&A]
I am trying to access the current snow level data, specifically at https://api.weather.gov/gridpoints/BOU/45,74. 

The get request returns the following:
```"snowLevel": {
            "uom": "wmoUnit:m",
            "values": [
                {
                    "validTime": "2021-04-03T08:00:00+00:00/PT1H",
                    "value": 2536.5455999999999
                },
```

I looked up the unit of measurement at: https://codes.wmo.int/common/unit/_m and this leads me to believe that the reported value is in meters. However, that would convert to 10,000 inches of snow. I have personally been to this location recently, and know that's not the case. It also seems excessively high even if I assume a typo and make it mm instead. So, how am I meant to interpret the value of snowLevel? I checked a couple of other gridpoints' reported snow level and the orders of magnitude are similar. 

Thank you for any assistance.

**Comment by jfbauer432** [comment]
I don't know for sure, but I suspect 2536 meters (8320 ft) might be the elevation that precipitation changes between rain and snow.  Or perhaps referring to snow on the ground at elevations above that.

**Comment by kelortondo** [comment]
A good thought @jfbauer432. I went and checked the elevation of the reporting gridpoint, and it is actually 2652 which is higher than the reported snow level, so I'm afraid that may not be it.
